.circleci/config.yml

```diff
@@ -75,8 +75,8 @@ commands:
           name: Install and configure dependencies
           command: |
             sudo npm install -g gh-pages@2.0.1
-            git config user.email "ci-build@binary.com"
-            git config user.name "ci-build"
+            git config user.email "mahboub.meng@gmail.com"
+            git config user.name "Mahboobeh-binary"
```

Comment:
is there any way to use a machine user github email account in deployment process? that would look good instead of using personal email account

Reply:
@shudarshon-binary yup. it was just used for test. will update it.